### DISCUSSION TITLE
Conditional fields: Multiple categories/entries with OR-like operator

### DISCUSSION THREAD

**Discussion by tsawitzki** [craftcms/cms#13890] [Ideas] [answered]
This is stemming from a support thread I had with Oli aiding me where no proper solution could be found, still it might be a realistic use case in certain setups:

I needed to check whether an entry is related to either two of three categories _A_, _B_ and _C_.
If _A_ or _B_ then show the fields, if _C_ do not show the fields.

With just one category ID it works fine, how can I use two though with an OR operator or comma separated values?
A simple OR between the IDs which should work in Twig doesn’t do it …

As Oli put it: „I spoke with Brandon and this is not possible at the moment. Even on element index, you can add multiple Related to conditions but those would only result in andRelatedTo again.“

Thanks in advance for considering this!

**Comment by brandonkelly** [comment, marked as answer]
Thanks for posting! This was previously requested here: #11287
